tests/q1_q1.prm

```diff
@@ -0,0 +1,97 @@
+############### Global parameters
```

Comment:
can you add a comment at the top along the lines of "test the q1q1 stabilization". Where did you copy the test from?